Mathlib/Topology/Algebra/ContinuousMonoidHom.lean

```diff
@@ -541,6 +542,28 @@ instance {M N} [Unique M] [Unique N] [Mul M] [Mul N]
 
 end unique
 
+/-- A family indexed by a type with a unique element
+is `ContinuousMulEquiv` to the element at the single index. -/
```

Comment:
Can you mention in the doc-string that this is the topological version of MulEquiv.piUnique resp. AddEquiv.piUnique?